src/test/tests-framework.js

```diff
@@ -0,0 +1,28 @@
+const {
+  startSourceMapWorker,
+  stopSourceMapWorker,
+} = require("devtools-source-map");
+
+const {
+  startPrettyPrintWorker,
+  stopPrettyPrintWorker,
+} = require("../utils/pretty-print");
+
+const {
+  startParserWorker,
+  stopParserWorker,
+} = require("../utils/parser");
+
+const { getValue } = require("devtools-config");
+
+beforeAll(() => {
+  startSourceMapWorker(getValue("workers.sourceMapURL"));
+  startPrettyPrintWorker(getValue("workers.prettyPrintURL"));
+  startParserWorker(getValue("workers.parserURL"));
+});
```

Comment:
I would re-use the bootstrapWorker function, but unfortunately, `bootstrap.js` pulls in `devtools-launchpad` which has `DebuggerConfig`. At some point it would be nice to use babel to replace `DebuggerConfig` with the appropriate config, but that is difficult given how jest uses babel.